webapp/src/components/sidebar_right/gitlab_items.tsx

```diff
@@ -124,9 +124,9 @@ function GitlabItems({item, theme}: GitlabItemsProps) {
             <SignIcon/>
             {item.milestone.title}
         </span>
-    ) : null;
+    ) : undefined;
 
-    const labels: JSX.Element[] | undefined = item.labels_with_details ? getGitlabLabels(item.labels_with_details) : null;
+    const labels: JSX.Element[] | undefined = item.labels_with_details ? getGitlabLabels(item.labels_with_details) : undefined;
```

Comment:
```suggestion
    const labels: JSX.Element[] | undefined = item.labels_with_details && getGitlabLabels(item.labels_with_details);
```
It will de undefined by default if you don't assign a value to it.